### ISSUE TITLE
UnicodeDecodeError: 'charmap' codec can't decode byte 0x90 in position 1152: character maps to <undefined>

### ISSUE THREAD

**Issue by LostInDarkMath** [goodmami/wn#95]
Hi there!
I run into an error while downloading `omw`:

```
>>> import wn
>>> wn.remove('odenet')
>>> wn.download(project_or_url='omw')
Download [##############################] (27701844/27701844 bytes) Complete

Traceback (most recent call last):
  File "<input>", line 1, in <module>
  File "D:\Projekte\fooGithub\source\backend_prototyp\venv\lib\site-packages\wn\_download.py", line 90, in download
    add_to_db(path, progress_handler=progress_handler)
  File "D:\Projekte\fooGithub\source\backend_prototyp\venv\lib\site-packages\wn\_add.py", line 82, in add
    for package in iterpackages(source):
  File "D:\Projekte\fooGithub\source\backend_prototyp\venv\lib\site-packages\wn\project.py", line 138, in iterpackages
    yield from iterpackages(contents[0])
  File "D:\Projekte\fooGithub\source\backend_prototyp\venv\lib\site-packages\wn\project.py", line 120, in iterpackages
    elif is_collection_directory(path):
  File "D:\Projekte\fooGithub\source\backend_prototyp\venv\lib\site-packages\wn\project.py", line 36, in is_collection_directory
    and len(list(filter(is_package_directory, path.iterdir()))) >= 1)
  File "D:\Projekte\fooGithub\source\backend_prototyp\venv\lib\site-packages\wn\project.py", line 29, in is_package_directory
    and len(list(filter(lmf.is_lmf, path.iterdir()))) == 1)
  File "D:\Projekte\fooGithub\source\backend_prototyp\venv\lib\site-packages\wn\lmf.py", line 321, in is_lmf
    xmldecl, doctype = _read_header(fh)
  File "D:\Projekte\fooGithub\source\backend_prototyp\venv\lib\site-packages\wn\lmf.py", line 328, in _read_header
    return (fh.readline().rstrip().replace("'", '"'),  # XML declaration
  File "C:\Users\WILLI\AppData\Local\Programs\Python\Python38\lib\encodings\cp1252.py", line 23, in decode
    return codecs.charmap_decode(input,self.errors,decoding_table)[0]
UnicodeDecodeError: 'charmap' codec can't decode byte 0x90 in position 1152: character maps to <undefined>
UnicodeDecodeError: 'charmap' codec can't decode byte 0x90 in position 1152: character maps to <undefined>
```

I'm using Python 3.8.0 and wn Version 0.5.0. How can I fix this?

**Comment by goodmami** [comment]
This looks like a classic opened-without-specifying-the-encoding problem. There are some places in the code where I'm opening files without specifying UTF-8, and Windows's default encoding is apparently something different. It's hard for me to test this, but let me see what I can do.

**Comment by goodmami** [comment]
I've released v0.5.1 which fixes the problem, I think. Can you please test it out and let me know how it goes?

```
python -m pip install -U wn
```

**Comment by LostInDarkMath** [comment]
Thanks for your very fast fix. Now it works fine :+1:
